vassal-app/src/main/java/VASSAL/counters/GlobalCommand.java

```diff
@@ -64,6 +65,9 @@
 
   private String fastProperty = "";     // Used during property Fast Match to hold *evaluated* expressions
   private String fastValue = "";        // Used during property Fast Match to hold *evaluated* expressions
+  private boolean fastIsNumber = false; // Used during property Fast Match to remember if value is numeric
+  private double fastNumber = 0;        // Used during property Fast Match to hold evaluated numerical value
+  private Pattern fastPattern = Pattern.compile(""); // Fast Match regex pattern
```

Comment:
Why compile a pattern that matches the empty string? (I'm surprised this doesn't throw.)

Reply:
I guess I was thinking there would be some annoying warning without it :) but happily there is not.